Description: Port to GStreamer 1.0 API.
Author: POJAR GEORGE <geoubuntu@gmail.com>
Bug: https://bugs.launchpad.net/ubuntu/+source/libgnome-media-profiles/+bug/1085118
Index: libgnome-media-profiles/configure
===================================================================
--- libgnome-media-profiles.orig/configure	2012-12-02 15:07:21.104163000 +0000
+++ libgnome-media-profiles/configure	2012-12-02 15:13:13.580542290 +0000
@@ -13172,12 +13172,12 @@
     if test -n "$PKG_CONFIG" && \
     { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"
     gtk+-3.0 >= \$GTK_REQUIRED_VERSION
-    gstreamer-0.10
+    gstreamer-1.0
     gconf-2.0
     gio-2.0\""; } >&5
   ($PKG_CONFIG --exists --print-errors "
     gtk+-3.0 >= $GTK_REQUIRED_VERSION
-    gstreamer-0.10
+    gstreamer-1.0
     gconf-2.0
     gio-2.0") 2>&5
   ac_status=$?
@@ -13185,7 +13185,7 @@
   test $ac_status = 0; }; then
   pkg_cv_GMP_CFLAGS=`$PKG_CONFIG --cflags "
     gtk+-3.0 >= $GTK_REQUIRED_VERSION
-    gstreamer-0.10
+    gstreamer-1.0
     gconf-2.0
     gio-2.0" 2>/dev/null`
 else
@@ -13200,12 +13200,12 @@
     if test -n "$PKG_CONFIG" && \
     { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"
     gtk+-3.0 >= \$GTK_REQUIRED_VERSION
-    gstreamer-0.10
+    gstreamer-1.0
     gconf-2.0
     gio-2.0\""; } >&5
   ($PKG_CONFIG --exists --print-errors "
     gtk+-3.0 >= $GTK_REQUIRED_VERSION
-    gstreamer-0.10
+    gstreamer-1.0
     gconf-2.0
     gio-2.0") 2>&5
   ac_status=$?
@@ -13213,7 +13213,7 @@
   test $ac_status = 0; }; then
   pkg_cv_GMP_LIBS=`$PKG_CONFIG --libs "
     gtk+-3.0 >= $GTK_REQUIRED_VERSION
-    gstreamer-0.10
+    gstreamer-1.0
     gconf-2.0
     gio-2.0" 2>/dev/null`
 else
@@ -13237,13 +13237,13 @@
         if test $_pkg_short_errors_supported = yes; then
 	        GMP_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors "
     gtk+-3.0 >= $GTK_REQUIRED_VERSION
-    gstreamer-0.10
+    gstreamer-1.0
     gconf-2.0
     gio-2.0" 2>&1`
         else
 	        GMP_PKG_ERRORS=`$PKG_CONFIG --print-errors "
     gtk+-3.0 >= $GTK_REQUIRED_VERSION
-    gstreamer-0.10
+    gstreamer-1.0
     gconf-2.0
     gio-2.0" 2>&1`
         fi
@@ -13252,7 +13252,7 @@
 
 	as_fn_error $? "Package requirements (
     gtk+-3.0 >= $GTK_REQUIRED_VERSION
-    gstreamer-0.10
+    gstreamer-1.0
     gconf-2.0
     gio-2.0) were not met:
 
Index: libgnome-media-profiles/configure.ac
===================================================================
--- libgnome-media-profiles.orig/configure.ac	2012-12-02 15:07:21.104163000 +0000
+++ libgnome-media-profiles/configure.ac	2012-12-02 15:10:54.484546425 +0000
@@ -29,7 +29,7 @@
 GTK_REQUIRED_VERSION=2.91.0
 PKG_CHECK_MODULES(GMP, [
     gtk+-3.0 >= $GTK_REQUIRED_VERSION
-    gstreamer-0.10
+    gstreamer-1.0
     gconf-2.0
     gio-2.0],
     have_profiles=yes)
@@ -57,9 +57,9 @@
 dnl Check for GStreamer
 dnl=======================================================================
 
-GST_REQ=0.10.23
-GST_PLUGINS_BASE_REQ=0.10.23
-GST_MAJORMINOR=0.10
+GST_REQ=1.0.3
+GST_PLUGINS_BASE_REQ=1.0.3
+GST_MAJORMINOR=1.0
 
 dnl make available to Makefile.am
 AC_SUBST(GST_MAJORMINOR)
Index: libgnome-media-profiles/data/gnome-media-profiles.schemas.in.in
===================================================================
--- libgnome-media-profiles.orig/data/gnome-media-profiles.schemas.in.in	2012-12-02 15:07:21.104163000 +0000
+++ libgnome-media-profiles/data/gnome-media-profiles.schemas.in.in	2012-12-02 15:12:11.768544127 +0000
@@ -125,7 +125,7 @@
       <applyto>/system/gstreamer/@GST_MAJORMINOR@/audio/profiles/cdlossy/pipeline</applyto>
       <owner>gnome-media</owner>
       <type>string</type>
-      <default>audio/x-raw-float,rate=44100,channels=2 ! vorbisenc name=enc quality=0.5 ! oggmux</default>
+      <default>audio/x-raw,rate=44100,channels=2 ! vorbisenc name=enc quality=0.5 ! oggmux</default>
       <locale name="C">
         <short>The partial GStreamer pipeline used</short>
         <long>
@@ -199,7 +199,7 @@
       <applyto>/system/gstreamer/@GST_MAJORMINOR@/audio/profiles/cdlossless/pipeline</applyto>
       <owner>gnome-media</owner>
       <type>string</type>
-      <default>audio/x-raw-int,rate=44100,channels=2 ! flacenc name=enc</default>
+      <default>audio/x-raw,rate=44100,channels=2 ! flacenc name=enc</default>
       <locale name="C">
         <short>The partial GStreamer pipeline used</short>
         <long>
@@ -274,7 +274,7 @@
       <applyto>/system/gstreamer/@GST_MAJORMINOR@/audio/profiles/voicelossless/pipeline</applyto>
       <owner>gnome-media</owner>
       <type>string</type>
-      <default>audio/x-raw-int,rate=22050,channels=1 ! wavenc name=enc</default>
+      <default>audio/x-raw,rate=22050,channels=1 ! wavenc name=enc</default>
       <locale name="C">
         <short>The partial GStreamer pipeline used</short>
         <long>
@@ -348,7 +348,7 @@
       <applyto>/system/gstreamer/@GST_MAJORMINOR@/audio/profiles/voicelossy/pipeline</applyto>
       <owner>gnome-media</owner>
       <type>string</type>
-      <default>audio/x-raw-int,rate=32000,channels=1 ! speexenc name=enc ! oggmux</default>
+      <default>audio/x-raw,rate=32000,channels=1 ! speexenc name=enc ! oggmux</default>
       <locale name="C">
         <short>The partial GStreamer pipeline used</short>
         <long>
@@ -422,7 +422,7 @@
       <applyto>/system/gstreamer/@GST_MAJORMINOR@/audio/profiles/mp2/pipeline</applyto>
       <owner>gnome-media</owner>
       <type>string</type>
-      <default>audio/x-raw-int,rate=44100,channels=2 ! twolame name=enc mode=0 bitrate=192 ! id3v2mux</default>
+      <default>audio/x-raw,rate=44100,channels=2 ! twolame name=enc mode=0 bitrate=192 ! id3v2mux</default>
       <locale name="C">
         <short>The partial GStreamer pipeline used</short>
         <long>
@@ -496,7 +496,7 @@
       <applyto>/system/gstreamer/@GST_MAJORMINOR@/audio/profiles/mp3/pipeline</applyto>
       <owner>gnome-media</owner>
       <type>string</type>
-      <default>audio/x-raw-int,rate=44100,channels=2 ! lamemp3enc name=enc target=0 quality=6 ! xingmux ! id3v2mux</default>
+      <default>audio/x-raw,rate=44100,channels=2 ! lamemp3enc name=enc target=0 quality=6 ! xingmux ! id3v2mux</default>
       <locale name="C">
         <short>The partial GStreamer pipeline used</short>
         <long>
@@ -570,7 +570,7 @@
       <applyto>/system/gstreamer/@GST_MAJORMINOR@/audio/profiles/aac/pipeline</applyto>
       <owner>gnome-media</owner>
       <type>string</type>
-      <default>audio/x-raw-int,rate=44100,channels=2 ! faac profile=2 ! ffmux_mp4</default>
+      <default>audio/x-raw,rate=44100,channels=2 ! faac profile=2 ! ffmux_mp4</default>
       <locale name="C">
         <short>The partial GStreamer pipeline used</short>
         <long>
